shared/src/test/diff/codegen/Mixin.mls

```diff
@@ -256,7 +256,7 @@ TestLang.eval(mk(0))
 //│ res = TestLang.eval(mk(0));
 //│ // End of generated code
 //│ res
-//│     = [Function: mk]
```

Comment:
I guess the `:js` flag for this block is to see why `res` is `mk`. Right? If so, I will remove `:js` flag.

Reply:
Yeah probably. Let's remove it!